cmd/esc/cli/client/client.go

```diff
@@ -677,6 +685,16 @@ func (pc *client) GetOpenEnvironmentWithProject(ctx context.Context, orgName, pr
 	return &resp, nil
 }
 
+func (pc *client) GetAnonymousOpenEnvironment(ctx context.Context, orgName, openSessionID string) (*esc.Environment, error) {
+	var resp esc.Environment
+	path := fmt.Sprintf("/api/esc/environments/%v/yaml/open/%v", orgName, openSessionID)
```

Comment:
Should this include project in the URL? As far as I understand, envName and project are not used, but eventually we will remove routes without project, right?

Also, nit - ideally should be more specific and use %s for strings, generic is always more bug-prone

Reply:
we will remove preview routes but this is a GA route. we decided that we don't need to include projects in this route for anonymous envs.